conans/model/build_info.py

```diff
@@ -228,6 +228,13 @@ def get_name(self, generator, default_name=True):
         return self.get_property(property_name, generator) \
                or self.names.get(generator, self._name if default_name else None)
 
+    # To provide compatibility for legacy generators with cmake_target_namespace
+    # and make migration easier
+    def get_namespace(self, generator):
+        if generator == "cmake_find_package" or "cmake_find_package_multi":
```

Comment:
This seems a bug, I guess you want to do:
```suggestion
        if generator == "cmake_find_package" or generator == "cmake_find_package_multi":
```
If a bug, check why it didn't fail in tests.

Reply:
Yes, sorry 🤦‍♂️
It's ok that it did not fail, this is just to filter who will listen to the property with the old cpp_info. It always entered the line where namespace was assigned but returning None if it was not declared.